tests/conftest.py

```diff
@@ -41,3 +45,29 @@ def cdn_test_url(request):
     else:
         pytest.skip("Test skipped without --cdn-test-url or --lambda-stack")
     return url
+
+
+def mock_conf_file():
+    temp_file = tempfile.NamedTemporaryFile(delete=False)
+
+    os.environ["PROJECT"] = "test"
```

Comment:
Hmm, I don't think you should dump these into the global os.environ here.  Note these stay in place for the whole test.

I think this generates confusion as the code under test is going to see env vars such as ORIGIN_RESPONSE_LOGGER_LEVEL which will never be there in production.  Someone might be wrongly led to believe that these env vars are present when the code runs, and even write code which depends on that with the tests wrongly passing.

subprocess.run takes an `env` argument which sets the environment for just that process, you should probably be using that so the env vars don't pollute all the tests.

EXODUS_LAMBDA_CONF_FILE is the only exception to this as that one really needs to be set for the code to work.